src/core/text/Text.js

```diff
@@ -196,8 +198,9 @@ Object.defineProperties(Text.prototype, {
 
             style.dropShadow = style.dropShadow || false;
             style.dropShadowColor = style.dropShadowColor || '#000000';
-            style.dropShadowAngle = style.dropShadowAngle || Math.PI / 6;
-            style.dropShadowDistance = style.dropShadowDistance || 5;
+            style.dropShadowAngle = style.dropShadowAngle !== undefined ? style.dropShadowAngle : Math.PI / 6;
+            style.dropShadowDistance = style.dropShadowDistance !== undefined ? style.dropShadowDistance : 5;
+            style.dropShadowBlur = style.dropShadowBlur !== undefined ? style.dropShadowBlur : 0; //shadowBlur is '0' by default according to HTML
```

Comment:
Probably not really part of this PR, but since I just saw this, thought I would comment that it would be great if this just used `Object.assign` here:
```js
this._style = Object.assign({}, Text.defaultStyle, style);
```
Yes, `Text.defaultStyle` does not exist just yet, but you get the idea.